src/main/java/gui/ShotBlock.java

```diff
@@ -101,6 +102,34 @@ public void setEndCount(double count) {
     }
 
     /**
+     * Set name of this shotblock. No Lombok because it does some extra work for the GUI.
```

Comment:
The reason this setter is not generated by Lombok does not have to be mentioned explicitly.